docs/reference/content/getting-started/quick-tour.md

```diff
@@ -303,6 +301,50 @@ myDoc = collection.find().projection(excludeId()).first();
 System.out.println(myDoc.toJson());
 ```
 
+## Updating documents
+
+There are numerous [update operators](http://docs.mongodb.org/manual/reference/operator/update-field/)
+supported by MongoDB.
+
+To update at most a single document, use the [`updateOne`]({{< apiref "com/mongodb/client/MongoCollection.html#updateOne-org.bson.conversions.Bson-org.bson.conversions.Bson-">}})
+method to specify a filter and the update document.  Here we update the first document that meets the filter `i` equals `10` and set the value of `i` to `110`:
+
+```java
+collection.updateOne(eq("i", 10), new Document("$set", new Document("i", 110)));
+```
+
+To update all documents matching the filter use the [`updateMany`]({{< apiref "com/mongodb/async/client/MongoCollection.html#updateMany-org.bson.conversions.Bson-org.bson.conversions.Bson-">}})
+method.  Here we increment the value of `i` by `100` where `i` is less than 100.
+
+```java
+UpdateResult updateResult = collection.updateMany(lt("i", 100),
+          new Document("$inc", new Document("i", 100)));
+System.out.println(updateResult.getModifiedCount());
+```
+
+The update methods return an [`UpdateResult`]({{< apiref "com/mongodb/client/result/UpdateResult.html">}})
+which provides information about the operation including the number of documents modified by the update.
+
+## Deleting documents
+
+To delete at most a single document (may be 0 if none match the filter) use the [`deleteOne`]({{< apiref "com/mongodb/client/MongoCollection.html#deleteOne-org.bson.conversions.Bson-">}})
```

Comment:
that parenthetical I intended for you, not for the docs, but I actually like it.  For consistency, add it to updateOne as well.